### ISSUE TITLE
cmd/go: failed to remove file (Access is denied) after child process on Windows in TestLocalImportsEasySub

### ISSUE THREAD

**Issue by bradfitz** [golang/go#23171]
Windows XP test flake:

https://build.golang.org/log/9f8d95ce0a594f5634d30e4f9b842decdd39ed3e
```
--- FAIL: TestLocalImportsEasySub (0.50s)
	go_test.go:1146: running testgo [build -o ./easysub.exe testdata\local\easysub\main.go]
	go_test.go:1147: remove ./easysub.exe: Access is denied.
FAIL
FAIL	cmd/go	90.989s
```

I looked at the test and it seems fine. It should wait for any child process to finish before it cleans up, so I don't see how windows would return an access error on the delete.

Maybe related to our general os.Process.Wait woes on Windows?

/cc @johnsonj @alexbrainman

**Comment by alexbrainman** [comment]
> Maybe related to our general os.Process.Wait woes on Windows?

Looks very likely the same problem to me.

Alex

**Comment by aclements** [comment]
I've been seeing this fairly regularly on the trybots. Just this morning:

https://go-review.googlesource.com/c/go/+/147098/7#message-504472723e1be98173b0abcc51dd8e752b9c7654
https://go-review.googlesource.com/c/go/+/146999/6#message-48d326d64319c8407540fe22ff0435d018981c17

**Comment by bradfitz** [comment]
@aclements, we removed (in 5c359736f8d67338b53c26aaef52139ae8cd0538, https://go-review.googlesource.com/c/145221) a 5ms sleep in the Windows exec wait path, hoping it was only needed on Windows XP, but apparently not. (Bug: #25965)

We're clearly doing something wrong on Windows if we need a 5ms sleep to work around it.

**Comment by bcmills** [comment]
> our general os.Process.Wait woes on Windows?

Is that #19491?

**Comment by youngjuning** [comment]
same issues

**Comment by bcmills** [comment]
`TestLocalImportsEasySub` no longer exists, and the other `cmd/go` tests should have been fixed with #30789. Closing this issue.

The underlying `Access is denied` failure still exists, and is tracked in #25965 and #50051.

**Comment by bcmills** [comment]
Duplicate of #30789
